### DISCUSSION TITLE
Setting up Xero integration with private bank / business account - incorrect wording on config page

### DISCUSSION THREAD

**Discussion by adejongh** [Investec-Developer-Community/.github#13] [General]
The wording on the "Manage" / "External Connections" / "Xero" web page where you link your Xero account is misleading, and this led me to believe that the integration was broken since I could never link an *existing* Xero account.  See this screen below:
![image](https://github.com/Investec-Developer-Community/.github/assets/8489747/6374dd00-a2ae-4118-8fbe-40c76a88c89b)

My existing Xero account(s) were never listed under the "Organization | Account name | Connection date | Status" row which led me to believe something was wrong in the Xero integration setup.

If you hover over the "Create a new Xero account" it tells you this:
<img width="330" alt="image" src="https://github.com/Investec-Developer-Community/.github/assets/8489747/7390513a-9ddb-45d1-98b3-b3396c2080da">

This *does* not seem like the right option to link an existing Xero account.  However, it is!

This configuration screen urgently needs some changing, as it does not make sense at all to use the "Create a new Xero account" to be able to connect an *existing* Xero account.

**Comment by programmable-banking-community** [comment]
That's a great point, thanks for the feedback! We'll pass this onto the CXT digital team.

**Comment by programmable-banking-community** [comment]
Hi, just to keep you updated; this is still on the Product team's radar and they have it on their backlog to resolve asap. Hope this helps for now.
